src/main/java/org/betonquest/betonquest/BetonQuest.java

```diff
@@ -302,18 +305,8 @@ public class BetonQuest extends JavaPlugin {
      */
     private final Map<String, QuestEventFactory> eventTypes = new HashMap<>();
     private final ConcurrentHashMap<Profile, PlayerData> playerDataMap = new ConcurrentHashMap<>();
```

Comment:
```suggestion
    private final Map<String, QuestEventFactory> eventTypes;
    private final ConcurrentHashMap<Profile, PlayerData> playerDataMap;
```

should be instantiated in the constructor as java conventions tells

Reply:
While I do agree with your statement in general, you are referencing code that was not changed in this PR and thus is irrelevant in a review.

You can of course create a separate PR to contribute this change.